### DISCUSSION TITLE
keybinding for vterm

### DISCUSSION THREAD

**Discussion by emacsbliss** [abougouffa/minemacs#134] [Q&A]
I'm trying to bind `C-l` to function `vterm-send-right` with following (on top of the latest version from repo):

```
  :config
  (+imap! :keymaps 'vterm-mode-map
    "C-l" #'vterm-send-right)
```

but it never works, not sure why, please help, thanks!

**Comment by abougouffa** [comment]
@emacsbliss 

Sorry, I didn't see this question till now. It should work! I've pushed a commit 58198b871e9a5d343f6b31ff76d3b895df46b7ed to use this by default.

**Reply by abougouffa** [reply]
It's worth noting that when defining a key map with `+imap!`, it **only works in insert mode**, not in normal mode. The `i` in `imap` stands for _insert_.
